docs/scripts/generate-llms/eas-docs.js

```diff
@@ -0,0 +1,36 @@
+import fs from 'node:fs';
+import path from 'node:path';
+
+import {
+  OUTPUT_DIRECTORY_NAME,
+  OUTPUT_FILENAME_EAS_DOCS,
+  TITLE_EAS,
+  DESCRIPTION_EAS,
+  generateSectionMarkdown,
+  processSection,
+} from './utils.js';
+import { eas } from '../../constants/navigation.js';
+
+function generateFullMarkdown({ title, description, sections }) {
+  return `# ${title}\n\n${description}\n\n` + sections.map(generateSectionMarkdown).join('');
+}
+
+export async function generateEasDocs() {
```

Comment:
```suggestion
export async function generateLlmsEasTxt() {
```

Let's also rename file to match.